pkg/api/v1alpha1/router.go

```diff
@@ -566,6 +566,45 @@ func (r *Router) Routes(rg *gin.RouterGroup) {
 		r.mwUserAuthRequired(AuthRoleAdmin),
 		r.deleteNotificationTarget,
 	)
+
+	// extensions
+	rg.GET(
+		"/extensions",
+		r.AuditMW.AuditWithType("ListExtensions"),
+		r.AuthMW.AuthRequired(readScopesWithOpenID("governor:extensions")),
+		r.listExtensions,
+	)
+
+	rg.GET(
+		"/extensions/:eid",
```

Comment:
I think everywhere else we use just `:id`

Reply:
I think using `:id` would cause a conflict with
`/extensions/:eid/erds/:erd-id`
which would be used for managing extension resource definitions